types/chayns/index.d.ts

```diff
@@ -0,0 +1,938 @@
+// Type definitions for chayns 0.14
+// Project: https://gitlab.tobit.com/chayns-api/chayns-typings
```

Comment:
the link seems invalid, is this the correct link: https://github.com/TobitSoftware/chayns-js?